sdk/nodejs/api/client.gen.ts

```diff
@@ -252,6 +252,8 @@ export class CacheVolume extends BaseClient {
 export class Container extends BaseClient {
   /**
    * Initialize this container from a Dockerfile build
+   *
+   * @param opts optional params for build
```

Comment:
Is there a real benefit in terms of DX to add this @param ? Do you think it could be possible to display an exhaustive list of those opts params ?

Reply:
I changed my opinion about it. It is not standard, but it would help.
I suggest: `@param opts.<field> ipsem lorem`
I'll add it

Reply:
Those decorators are only useful if you generate doc from source code using [JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) for example.
I do not find it useful there since the field is already named. 
I think you can stick either to `opts.<field> lorem` or just `<fields> lorem`.

Reply:
> Those decorators are only useful if you generate doc from source code using [JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) for example. I do not find it useful there since the field is already named. I think you can stick either to `opts.<field> lorem` or just `<fields> lorem`.

But those decorators are also used by typedoc : https://typedoc.org/tags/param/#object-literals
Even if it works without it, I think we semantically need them, does it make sense ?

Reply:
You're right!